StepDefinitions/HotelsSD/HotelRemoveGusSD.js

```diff
@@ -0,0 +1,121 @@
+const { Given, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const HotelHomePage = require("../../Pages/Hotels/HotelHomePage");
+
+const hotelhomepage = new HotelHomePage();
+
+Given(/^I am on hotels homepage$/, async function() {
+    await browser.url('/');
+    await browser.pause(1000);
+})
+
+Then(/^I click on Travelers$/, async function() {
+    await hotelhomepage.clickTravelersBox();
+    await browser.pause(2000);
+})
```

Comment:
I saw you implemented this step on StepDefinitions/HotelsSD/HotelAddGusSD.js file. Once you implemented one step you don't have to implement it again just use it. Thats the beauty of Cucumber.